app/scripts/index.js

```diff
@@ -175,7 +176,6 @@ app.run(function ($rootScope, $location, $anchorScroll, $window) {
         // Put updated object into storage
         localStorage.setItem('consoleSession', JSON.stringify(consoleSessionObject))
```

Comment:
Let's remove all code about frontend session handling.

Reply:
I removed loggedOnUser session and kept this one, the one that I removed is not used